qa/L0_s3_local/test.sh

```diff
@@ -93,56 +97,60 @@ awslocal $ENDPOINT_FLAG s3 rm s3://demo-bucket1.0 --recursive --include "*" && \
 
 # Create and add data to bucket
 awslocal $ENDPOINT_FLAG s3 mb s3://demo-bucket1.0 && \
-    awslocal $ENDPOINT_FLAG s3 sync $MODELDIR s3://demo-bucket1.0
+    awslocal $ENDPOINT_FLAG s3 sync models s3://demo-bucket1.0
 
 RET=0
 
-# Test with hostname
-SERVER_ARGS="--model-repository=s3://localhost:4572/demo-bucket1.0 --model-control-mode=explicit"
-SERVER_LOG="./inference_server_hostname.log"
-
-BACKENDS="graphdef libtorch onnx plan savedmodel"
+# Test with hostname and IP address
+for HOST in "127.0.0.1" "localhost"; do
+    SERVER_ARGS="--model-repository=s3://$HOST:4572/demo-bucket1.0 --model-control-mode=explicit"
+    if [ "$HOST" = "127.0.0.1" ]; then
+        SERVER_LOG="./inference_server_hostname.log"
+    else
+        SERVER_LOG="./inference_server_ip.log"
+    fi
 
-run_server
-if [ "$SERVER_PID" == "0" ]; then
-    echo -e "\n***\n*** Failed to start $SERVER\n***"
-    cat $SERVER_LOG
-    # Kill minio server
-    kill $MINIO_PID
-    wait $MINIO_PID
-    exit 1
-fi
+    run_server
+    if [ "$SERVER_PID" == "0" ]; then
+        echo -e "\n***\n*** Failed to start $SERVER\n***"
+        cat $SERVER_LOG
+        # Kill minio server
+        kill $MINIO_PID
+        wait $MINIO_PID
+        exit 1
+    fi
 
-set +e
-for BACKEND in $BACKENDS; do
-    code=`curl -s -w %{http_code} -X POST localhost:8000/v2/repository/models/${BACKEND}_float32_float32_float32/load`
+    set +e
+    for BACKEND in $BACKENDS; do
+        code=`curl -s -w %{http_code} -X POST localhost:8000/v2/repository/models/${BACKEND}_float32_float32_float32/load`
+        if [ "$code" != "200" ]; then
+            echo -e "\n***\n*** Test Failed\n***"
+            RET=1
+        fi
+
+        $PERF_CLIENT -m ${BACKEND}_float32_float32_float32 -p 3000 -t 1 >$CLIENT_LOG 2>&1
+        if [ $? -ne 0 ]; then
+            echo -e "\n***\n*** Test Failed\n***"
+            cat $CLIENT_LOG
+            RET=1
+        fi
+    done
+
+    # Try to load model with name that checks for all types of allowed characters
+    code=`curl -s -w %{http_code} -X POST localhost:8000/v2/repository/models/${DUMMY_MODEL}/load`
     if [ "$code" != "200" ]; then
         echo -e "\n***\n*** Test Failed\n***"
         RET=1
     fi
+    set -e
 
-    $PERF_CLIENT -m ${BACKEND}_float32_float32_float32 -p 3000 -t 1 >$CLIENT_LOG 2>&1
-    if [ $? -ne 0 ]; then
-        echo -e "\n***\n*** Test Failed\n***"
-        cat $CLIENT_LOG
-        RET=1
-    fi
+    kill $SERVER_PID
+    wait $SERVER_PID
 done
 
-# try to load model with name that checks for all types of allowed characters
-code=`curl -s -w %{http_code} -X POST localhost:8000/v2/repository/models/${DUMMY_MODEL}/load`
-if [ "$code" != "200" ]; then
-    echo -e "\n***\n*** Test Failed\n***"
-    RET=1
-fi
-set -e
-
-kill $SERVER_PID
-wait $SERVER_PID
-
-# Test with IP
-SERVER_ARGS="--model-repository=s3://127.0.0.1:4572/demo-bucket1.0 --model-control-mode=explicit"
-SERVER_LOG="./inference_server_ip.log"
+# Test with Polling
```

Comment:
Should test the same in explicit mode

Reply:
Already tested in explicit mode in L0_storage_S3